crates/common/consensus/lean/src/attestation.rs

```diff
@@ -1,18 +1,26 @@
+use alloy_primitives::FixedBytes;
 use serde::{Deserialize, Serialize};
 use ssz_derive::{Decode, Encode};
 use ssz_types::{BitList, VariableList, typenum::U4096};
 use tree_hash_derive::TreeHash;
 
-use crate::vote::Vote;
+use crate::vote::AttestationData;
 
-/// Represents an attestation in the Lean chain.
+/// Aggregated attestation consisting of participation bits and message.
 ///
 /// See the [Lean specification](https://github.com/leanEthereum/leanSpec/blob/main/docs/client/containers.md#attestation)
 /// for detailed protocol information.
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash)]
-pub struct Attestation {
+pub struct AggregatedAttestation {
```

Comment:
I think this should be `AggregatedAttestations` similar to `AggregatedSignatures`. I've left a comment in the spec PR, let's see.